cmake/EmbedResourceTargets.cmake

```diff
@@ -106,13 +106,10 @@ function(_target_add_resource target name)
 
     target_sources(${target} PRIVATE ${out_f})
     target_include_directories(${target} PUBLIC "${EMBEDRESOURCE_INCLUDE_DIR}")
-    target_compile_definitions(${target} PUBLIC HAVE_EMBEDRESOURCE=1)
 endfunction()
 
 function(add_resource_library)
```

Comment:
you will have to change the ci/sample/CMakeLists.txt to reflect the change in argument to fix the tests